src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-download-dialog/content-download-dialog.module.ts

```diff
@@ -17,8 +17,5 @@ import { DialogModule } from '../dialog/dialog.module';
   exports: [
     ContentDownloadDialogComponent,
   ],
-  entryComponents: [
```

Comment:
Test if entry components are breaking on run time in content portal and excel app